src/hooks/useIsByTransactionParams.ts

```diff
@@ -1,11 +1,13 @@
-import { useMemo } from 'react';
-import { useSearchParams } from 'react-router-dom';
-
+import { useMemo } from "react";
```

Comment:
On a side note: this hook has a very odd name, what is "IsByTransactionParams" supposed to mean?

Might flag this for a rename